content/influxdb/v2.0/reference/cli/influx/task/retry-failed.md

```diff
@@ -0,0 +1,92 @@
+---
+title: influx task retry-failed
+description: The `influx task retry-failed` command retries failed InfluxDB a task runs.
+menu:
+  influxdb_2_0_ref:
+    name: influx task retry-failed
+    parent: influx task
+weight: 201
+---
+
+The `influx task retry-failed` command retries failed InfluxDB a task runs.
+
+## Usage
+```
+influx task retry-failed [flags]
+```
+
+## Flags
+| Flag |                   | Description                                                               | Input type  | {{< cli/mapped >}}    |
+|:---- |:---               |:-----------                                                               |:----------: |:------------------    |
+|      | `--after`         | Retry task runs that occurred after this time (RFC3339 timestamp)         | string      |                       |
+| `-c` | `--active-config` | CLI configuration to use for command                                      | string      |                       |
+|      | `--configs-path`  | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`)     | string      |`INFLUX_CONFIGS_PATH`  |
+|      | `--before`        | Retry task runs that occurred before this time (RFC3339 timestamp)        | string      |                       |
+|      | `--dry-run`       | Print information about runs that would be retried                        |             |                       |
+| `-h` | `--help`          | Help for the `list` command                                               |             |                       |
+|      | `--hide-headers`  | Hide table headers (default `false`)                                      |             | `INFLUX_HIDE_HEADERS` |
+|      | `--host`          | HTTP address of InfluxDB (default `http://localhost:8086`)                | string      | `INFLUX_HOST`         |
+| `-i` | `--id`            | Task ID                                                                   | string      |                       |
+|      | `--json`          | Output data as JSON (default `false`)                                     |             | `INFLUX_OUTPUT_JSON`  |
+| `-o` | `--org`           | Task organization name                                                    | string      | `INFLUX_ORG`          |
+|      | `--org-id`        | Task organization ID                                                      | string      | `INFLUX_ORG_ID`       |
+|      | `--run-limit`     | Maximum number of failed runs to retry per task (`1-500`, default `100`)  | integer     |                       |
+|      | `--skip-verify`   | Skip TLS certificate verification                                         |             |                       |
+|      | `--task-limit`    | Maximum number of tasks to retry failed runs for (`1-500`, default `100`) | integer     |                       |
```

Comment:
Pedantic, yes, but a bit clearer, maybe 😇 :

```suggestion
|      | `--task-limit`    | Maximum number of tasks for which to retry failed runs (`1-500`, default `100`) | integer     |                       |
```

Reply:
Max # of tasks to retry at the same time?

Reply:
https://developers.google.com/style/prepositions ...I tend away from "preposition+which" :-) but agree perhaps rewording somehow